crypto/rand/drbg_lib.c

```diff
@@ -127,19 +132,26 @@ static RAND_DRBG *rand_drbg_new(int secure,
                                 RAND_DRBG *parent);
 
 /*
- * Set/initialize |drbg| to be of type |nid|, with optional |flags|.
+ * Set/initialize |drbg| to be of type |type|, with optional |flags|.
+ *
+ * If |type| and |flags| are zero, use the defaults
  *
  * Returns 1 on success, 0 on failure.
  */
-int RAND_DRBG_set(RAND_DRBG *drbg, int nid, unsigned int flags)
+int RAND_DRBG_set(RAND_DRBG *drbg, int type, unsigned int flags)
```

Comment:
I'm currently wondering why the nid/type is a `int` and whether it should be changed to an `unsigned int` before the beta freeze.

Any thoughts or opinions?

Reply:
Perhaps because it's an `int` everywhere else?  I'm not sure it matters, since they are constant numbers and we only ever check for equality (right?)

Reply:
You're right. The fact that NID's are passed as an `int` everywhere else convinces me to leave it as it is.